lib/scrabble.rb

```diff
@@ -1,7 +1,33 @@
+require 'pry'
 class Scrabble
 
   def score(word)
-    1
+    word = nil_value(word)
```

Comment:
It's good that this method moves the logic out, but we should want to stop if the incoming parameter is nil or an empty string.